src/main/java/de/tum/in/www1/artemis/repository/ExerciseRepository.java

```diff
@@ -260,42 +289,33 @@ SELECT COUNT(DISTINCT p.student.id)
             FROM Exercise e JOIN e.studentParticipations p
             WHERE e.id = :exerciseId
             """)
-    Long getParticipationCountById(@Param("exerciseId") Long exerciseId);
+    Long getStudentParticipationCountById(@Param("exerciseId") Long exerciseId);
 
     /**
-     * Fetches the score of the given exercise
+     * Fetches the amount of team participations in the given exercise
      *
-     * @param exerciseId the id of the exercise to get the score for
-     * @return The average score as <code>Double</code>
+     * @param exerciseId the id of the exercise to get the amount for
+     * @return The amount of participations as <code>Long</code>
```

Comment:
I think from a wording point of view, 'number of participations' might be more fitting, but that's just a minor thing

Reply:
Done.